core/arch/arm/kernel/entry_a64.S

```diff
@@ -338,6 +338,7 @@ clear_nex_bss:
 	str	wzr, [x22, #THREAD_CORE_LOCAL_FLAGS]
 #endif
 	mov	x0, x20		/* DT address */
+	mov	x1, x19		/* TOS_FW_CONFIG DT address */
```

Comment:
If not `CFG_CORE_SEL1_SPMC` then we should supply zeroes in `x1` and vice versa in `x0` above where `boot_init_primary_early()` is called with pageable part address.
`boot_init_primary_late()` is called form `entry_a32.S` too, there we must make sure to supply zeroes in `r1`.

Reply:
Fixed